ci/lnd-sidecar-image/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.15.0
```

Comment:
is it possible to have a distroless image?

moving every Docker image to a distroless container would also improve the security

Reply:
I tried to have the distroless be the base image, but ran into issues with shared libraries that were missing from the distroless image but required by our dependencies.